packages/core/src/di/contextual.ts

```diff
@@ -47,6 +48,12 @@ export function runInInjectionContext<ReturnT>(injector: Injector, fn: () => Ret
   }
 }
 
+/**
+ * Whether the current stack frame is inside an injection context.
+ */
+export function isInInjectionContext() {
```

Comment:
```suggestion
export function isInInjectionContext(): boolean {
```

This will catch a type error, as the inferred return type is not `boolean`, but a union type of the results of the two functions.

Reply:
Fixed. Meant to do that— but in practice didn't affect anything in the TGP